build/sdk-test-env.bat

```diff
@@ -1,21 +0,0 @@
-@echo off
```

Comment:
I think my comment about these still being useful was lost in the original PR. Can we please fix up the paths and retain these?

Reply:
Added a `-dogfood` option and an explicit `build\dogfood.bat` script